book_source/02_basic_users_guide/02_web_workflow/00_web_workflow.Rmd

```diff
@@ -1,9 +1,3 @@
 # Web workflow
 
-- Model and site selection
-- Run configuration
-- Run execution
-- Results
-- Interactive visualizations
```

Comment:
Yes, I forgot to add this one back. In general, I _don't_ like this idea because we have to maintain these manually, which is a pain and it's really easy for them to get out of date. But I'll oblige.

Reply:
So this outline is actually _wrong_ based on the sections currently in here. I added these headings back in, with some links where they apply, but marked the rest as `TODO`. I'll see if I can find them elsewhere in the documentation.